.github/actions/flaky-tests-report/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Flaky test report
+description: Extract flaky tests from PW test reports
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 20
+
+    - name: Install CTRF package globally
+      shell: bash
+      run: npm install -g ctrf
+
+    - name: Download CTRF reports
+      uses: actions/download-artifact@v4
+      with:
+        pattern: ctrf-report-*
+        path: ctrf-reports
+        merge-multiple: true
+        
+    - name: Merge CTRF reports
+      shell: bash
+      run: npx ctrf merge ./ctrf-reports --output merged-report.json
```

Comment:
Shouldn't we consider adding error handling or conditional checks in case of failure?
If this command fails (e.g., due to a missing ctrf-reports directory or invalid output location), the workflow could break without proper feedback 🤔

Reply:
@michalina-graczyk I think no need for extra error handling, the job is already failing if there are no files to merge, or merge report wasn't created here is an example. So as developer you will see that not all checks passed.
<img width="1777" alt="image" src="https://github.com/user-attachments/assets/8bbc17d8-4db9-49b7-8da5-3c6eebf432dd" />